src/main.rs

```diff
@@ -54,6 +61,42 @@ fn choose_target<'a>(args: &Arguments, manifest: &'a Manifest) -> Result<&'a Tar
     }
 }
 
+fn run_2018(args: &Arguments, manifest: &Manifest) -> Result<(), Error> {
+    // TODO: Check to see if build scripts really need to be ignored.
+    //       Seems like they are not mistaken as orphans anyway.
+    let build_scripts: Vec<path::PathBuf> = manifest
+        .custom_builds()
+        .iter()
+        .map(|t| t.src_path().clone())
+        .collect();
+    let target: &Target = choose_target(args, &manifest)?;
+    let include_orphans: bool = args.orphans;
+    let enable_color: bool = if cfg!(target_os = "windows") {
+        false
+    } else {
+        !args.plain
+    };
+    colored::control::set_override(enable_color);
```

Comment:
Once we do this, we don't need to keep doing `if color_enabled {...`

Reply:
That is indeed much nicer!